controllers/comments.controllers.js

```diff
@@ -1,9 +1,18 @@
-const { fetchCommentsByArticleId } = require("../models/comments.models");
+const { fetchCommentsByArticleId, insertComment } = require("../models/comments.models");
 
 exports.getCommentsByArticleId = (req, res, next) => {
     const { article_id } = req.params;
     fetchCommentsByArticleId(article_id).then((comments) => {
         res.status(200).send({ comments });
     })
     .catch(next);
+};
+
+exports.postComment = (req, res, next) => {
+    const { article_id } = req.params;
+    const newComment = req.body;
+    insertComment(article_id, newComment).then((postedComment) => {
+        res.status(201).send({ postedComment: postedComment.body })
```

Comment:
You should send back the entire comment object rather than just the body - this would be the usual behaviour when posting a resource to a RESTful api

Reply:
I have changed it to sending the entire object. :)